src/tree/tests/kat_tree_kem.rs

```diff
@@ -0,0 +1,252 @@
+//! TreeKEM test vectors
+//!
+//! See https://github.com/mlswg/mls-implementations/blob/master/test-vectors.md
+//! for more description on the test vectors.
+//!
+//! The test vector describes a tree of `n` leaves adds a new leaf with
+//! `my_key_package` and `my_path_secret` (common ancestor of `add_sender` and
+//! `my_key_package`).
+//! Then an update, sent by `update_sender` with `update_path` is processed, which
+//! is processed by the newly added leaf as well.
+//!
+//! Some more points
+//! * An empty group context is used.
+//! * update path with empty exclusion list.
+
+#[cfg(any(feature = "expose-test-vectors", test))]
```

Comment:
I'd expect this file only to be compiled for tests or test vectors. So why is this needed? (Same for the other `cfg` below.)